ftests/parallel/test_remote.py

```diff
@@ -129,15 +129,21 @@ def test_dsoln(self):
         from nose.plugins.skip import SkipTest
         if sys.platform.startswith('win'): raise SkipTest
         from numpy import zeros
-        from solvcon.batch import Localhost
         from solvcon.domain import Distributed
+        if "SOLVCON_NODELIST" in os.environ:
```

Comment:
I don't really make sure that a `if` statement is better, or just creating another test case e.g. `test_dsoln_local` and `test_dsoln_cluster`. `test_dsoln_cluster` will be skipped automatically if `SOLVCON_NODELIS` is not defined.